### ISSUE TITLE
Number of epochs ?

### ISSUE THREAD

**Issue by rajatsaini0294** [d-li14/mobilenetv2.pytorch#5]
For how many epochs you trained the network to get 72% accuracy ?

**Comment by rajatsaini0294** [comment]
Got it. Sorry for any inconvenience.
